boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -67,7 +67,7 @@ REGISTRY_USER ?=
 REGISTRY_TOKEN ?=
 
 BINFILE=build/_output/bin/$(OPERATOR_NAME)
-MAINPACKAGE ?= ./cmd/manager
+MAINPACKAGE ?= ./main.go
```

Comment:
Does this mean consuming repos will need to move their main.go and/or override this variable when they update to/past this level of boilerplate?

Reply:
yes, that's the plan so far, we are working on an step by step guide how to do the migration at the same time. 
I am also working on make boilerplate compatible by checking which osdk you are using.